test/Sector3Priority.ts

```diff
@@ -227,4 +227,16 @@ describe("Sector3DAOPriority", function () {
       expect(await sector3DAOPriority.getEpochIndex()).to.equal(1);
     });
   });
+
+  describe("addContribution", async function() {
+    it("Contributions array should be empty immediately after deployment", async function() {
+      const { sector3DAOPriority, owner } = await loadFixture(deployWeeklyFixture);
+
+      const epochIndex = await sector3DAOPriority.getEpochIndex();
+      console.log("epochIndex:", epochIndex);
+      // const contributions = await sector3DAOPriority.contributionsByEpochIndex(epochIndex);
```

Comment:
@solleedev I cannot understand why the `contributionsByEpochIndex()` function requires two parameters instead of one.  I expected only one `uint8` parameter, but it requires two:

From the ABI:
```json
...

"inputs": [
        {
          "internalType": "uint8",
          "name": "",
          "type": "uint8"
        },
        {
          "internalType": "uint256",
          "name": "",
          "type": "uint256"
        }
      ],
      "name": "contributionsByEpochIndex",
...
```

Any ideas? 😅